internals-js/src/operations.ts

```diff
@@ -1932,9 +1926,13 @@ class FragmentSpreadSelection extends FragmentSelection {
 export function operationFromDocument(
   schema: Schema,
   document: DocumentNode,
-  operationName?: string,
+  options?: {
```

Comment:
Note for reviewer: I didn't wanted to add the new `validate` as a nameless argument because that was starting to get ugly, especially when the `operationName` was already optional. But at the same time, `operationFromDocument` is called in a ton of places (particularly tests), almost always with just the 2 first arguments, so switching to _all_ argument being name-based was pretty annoying and would have generated a big diff. Hence why I opted for this compromised of keeping the 2 first main argument nameless and putting everything else in an "options" object.